File: 	1038.py
codigo, quantidade = input().split()
codigo = int(codigo)
quantidade = int(quantidade)
if codigo ==1:
    valor = quantidade*4.00
if codigo ==2:
  valor = quantidade*4.50
if codigo ==3:
  valor = quantidade*5.00
if codigo ==4:
  valor = quantidade*2.00
if codigo ==5:
  valor = quantidade*1.50
print("Total: R$ %.2f"%valor)

File: 	1038.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let [codigo, quantidade] = lines.shift().split(' ');
let conta =0;

if(codigo == 1){
    conta = quantidade* 4.0;
}

else if(codigo == 2){
     conta = quantidade* 4.5;
}

else if(codigo == 3){
     conta = quantidade* 5.0;
}

else if(codigo == 4){
     conta = quantidade* 2.0;
}

else if(codigo == 5){
     conta = quantidade* 1.5;
}
console.log(`Total: R$ ${conta.toFixed(2)}`);
